test/e2e/e2eutil/spec_util.go

```diff
@@ -78,8 +78,9 @@ func NewEtcdRestore(restoreName, version string, size int, restoreSource api.Res
 		},
 		Spec: api.RestoreSpec{
 			ClusterSpec: api.ClusterSpec{
-				Size:    size,
-				Version: version,
+				BaseImage: "gcr.io/etcd-development/etcd",
```

Comment:
what if version is 3.0.x? Will this code find the right image?

maybe we should also pass in the base image as a param.